src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_format.py

```diff
@@ -114,3 +114,11 @@ def get_vmss_table_output_transformer(loader, for_list=True):
     transform = transform.replace('$zone$', 'Zones: (!zones && \' \') || join(\' \', zones), '
                                   if loader.supported_api_version(min_api='2017-03-30') else ' ')
     return transform if not for_list else '[].' + transform
+
+
+def transform_vm_encryption_show_table_output(result):
+    from collections import OrderedDict
+    if "status" in result and result["status"]:
+        status_dict = result["status"][0]
+        return OrderedDict([("status", status_dict["displayStatus"]), ("message", status_dict["message"])])
```

Comment:
I suggest use `get` to avoid crash per nature of the unpredictability of the encryption extension. Suggest use default value of 'N/A'